collectfast/management/commands/collectstatic.py

```diff
@@ -32,14 +32,7 @@ def __init__(self, *args, **kwargs):
         self.tasks = []
         self.etags = {}
         self.collectfast_enabled = settings.enabled
-
-        if is_boto(self.storage) and self.storage.preload_metadata is not True:
-            self.storage.preload_metadata = True
-            warnings.warn(
-                "Collectfast does not work properly without "
-                "`preload_metadata` set to `True` on the storage class. Try "
-                "setting `AWS_PRELOAD_METADATA` to `True`. Overriding "
-                "`storage.preload_metadata` and continuing.")
+        self.storage_extensions = staticfiles_storage_extensions
```

Comment:
Since `staticfiles_storage_extensions` is lazy loaded, this code will trigger the storage class' `__init__()` function, which in turn triggers `check_preload_metadata()`, which contains this same code without the `is_boto(...)` statement